PhotoPoints/app/src/main/java/edu/cascadia/mobas/photopoints/repo/PhotoPointsRepository.java

```diff
@@ -66,7 +67,7 @@ private PhotoPoint mapSingle(DBPhotoPoint point){
         List<PhotoPoint> points = new ArrayList<>();
 
         for(DBPhotoPoint point : dbPoints){
-            points.add(mapSingle(point));
+            points.add(new PhotoPoint(point.getPhotoPointID(), point.getLatitude(), point.getLongitude(), point.getQRCode(), point.getPhotoPointType()));
```

Comment:
Why not us the mapSingle(point) so we don't have to change this code in multiple locations?

Reply:
Done